services/121-service/src/notifications/README.md

```diff
@@ -0,0 +1,43 @@
+## Testing Twilio API during development
+
+### 1. Use a tool to inspect the responses from the Twilio API.
+
+- Some options:
+  - `ngrok`: <https://ngrok.com>:
+    - See also: <https://www.twilio.com/en-us/blog/6-awesome-reasons-to-use-ngrok-when-testing-webhooks-html>
+    - Make sure to use the correct port(`3000`) of the 121-service.
+  - `Smee`: <https://smee.io/>
+    - You can use the client with:  
+      `npx smee -u https://smee.io/<unique-url>`
+  - Or any other service that gives a public accessible URL to inspect and/or forward to you local instance of the 121-service.
+- Setup steps with `ngrok`:
+  - Install ngrok on your machine, and link to a free public account
+  - Run `ngrok http 3000`
+- Make note of your publicly accessible URL, for example `https://srkgjh-33-444-88-111.ngrok-free.app`
+
+### 2. Login to the [Twilio console](https://console.twilio.com/us1/develop/sms/try-it-out/whatsapp-learn?frameUrl=%2Fconsole%2Fsms%2Fwhatsapp%2Flearn%3Fx-target-region%3Dus1) and link yourself to the sandbox
+
+- Make sure to choose the "development" subaccount from the dropdown menu at the top of the page
+- Make note of the `SID` and `AUTHTOKEN`. You'll need these later.
+- Access the "Sandbox" environment from the left menu: Messaging > Try it out > Send a WhatsApp Message
+- Make note of the `WHATSAPP_NUMBER` and then join the conversation with your personal WhatsApp account
+- In "Sandbox settings", update the callback URLs to use your publicle accessible URL
+  - For example, `https://srkgjh-33-444-88-111.ngrok-free.app/api/notifications/whatsapp/incoming`
+
+### 3. Update your `services/.env` file
+
+- Set `TWILIO_SID`, `TWILIO_AUTHTOKEN`, and `TWILIO_WHATSAPP_NUMBER` to the information gathered above
+- Set `MOCK_TWILIO=` (empty string)
+- Ask a colleague for the `TWILIO_MESSAGING_SID`
```

Comment:
- You can go to [messaging -> services](https://console.twilio.com/us1/develop/sms/services) to find the Sid of the messaging service (`TWILIO_MESSAGING_SID`)